tests/select_font.rs

```diff
@@ -19,6 +19,7 @@ use std::ffi::OsStr;
 #[cfg(feature = "source")]
 use font_kit::source::SystemSource;
 
+#[cfg(all(feature = "source", target_os = "windows"))]
```

Comment:
Looks like this cannot be Windows-only because it causes the MacOS build to fail.